tests/foreman/cli/test_syncplan.py

```diff
@@ -135,6 +135,17 @@ def validate_repo_content(repo, content_types, after_sync=True):
             assert not int(repo['content-counts'][content])
 
 
+def disable_syncplan(sync_plan):
```

Comment:
This is a teardown and you need not to perform it from CLI. You can use the api utils that you have (if) created as per above suggestion.

Reply:
How would you call the API version of `disable_syncplan()` helper from CLI test? The structure of sync_plan in CLI is different to that in the API.

Reply:
@swadeley Since the teardown is different from the actual test steps / test object, this is not set in stone to teardown the test fron CLI, you should be able to choose the easiest/faster way to do it. And API is always faster.

Reply:
@jyejare I have moved the helper from the API module to the api/utils file.

To get the CLI module to be able to use the API helper I added a line to read the sync plan using the nailgun code so that a sync plan object can be passed form the CLI module to the API helper in the expected format. I am sure there may be other ways to do this, e.g. pass org annd sync plan ID to the helper and do the read there, but then you are adding an extra read step in the case of API test.

Thank you.

Reply:
I have now added request.addfinalizer step to API, CLI, and UI modules.